pilz_control/test/acceptance_test_acceleration_limit.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# Copyright (c) 2019 Pilz GmbH & Co. KG
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import actionlib
+import rospy
+
+from actionlib_msgs.msg import *
+from control_msgs.msg import *
+from trajectory_msgs.msg import JointTrajectory, JointTrajectoryPoint
+
+COMMAND_TOPIC_NAME = '/prbt/manipulator_joint_trajectory_controller/command'
+FOLLOW_JOINT_TRAJ_ACTION_NAME = '/prbt/manipulator_joint_trajectory_controller/follow_joint_trajectory'
+JOINT_NAMES = ['prbt_joint_1', 'prbt_joint_2', 'prbt_joint_3', 'prbt_joint_4', 'prbt_joint_5', 'prbt_joint_6']
+
+DEFAULT_QUEUE_SIZE = 10
+DEFAULT_TRAJECTORY_DURATION_S = 5
+WAIT_FOR_SERVICE_TIMEOUT_S = 10
```

Comment:
why not make this a `rospy.Duration` from the start?

Reply:
I don't know. Why does it matter?

Reply:
resolved